pkg/test/ginkgo/external.go

```diff
@@ -29,6 +29,11 @@ type serializedTest struct {
 func externalTestsForSuite(ctx context.Context) ([]*testCase, error) {
 	var tests []*testCase
 
+	// TODO get client from real client location
```

Comment:
this looks weird, why?

Reply:
Probably a bad rebase :/

Reply:
oh, its a commit from https://github.com/openshift/origin/pull/28222 - https://github.com/openshift/origin/commit/3aec209b700d83ebf65dd48d281295fd06116d05.

Probably not necessary here